omniNotes/src/androidTest/java/it/feio/android/omninotes/helpers/Mathers.java

```diff
@@ -0,0 +1,60 @@
+package it.feio.android.omninotes.helpers;
+
+import android.support.test.espresso.UiController;
+import android.support.test.espresso.ViewAction;
+import android.support.test.espresso.matcher.ViewMatchers;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewParent;
+import android.widget.ListView;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+import static android.support.test.espresso.Espresso.onView;
+
+public class Mathers {
```

Comment:
The name of your class is not suggestive. You should try : "HelpersMethods" .

Reply:
And in this class you should have only the Espresso methods (eg: isVisible etc).